internal/pkg/deploy/cloudformation/cloudformation.go

```diff
@@ -164,3 +365,42 @@ func stackExists(err error) bool {
 	}
 	return false
 }
+
+// stackExists returns true if the underlying error is a stack already exists error.
+func stackSetExists(err error) bool {
+	if aerr, ok := err.(awserr.Error); ok {
+		switch aerr.Code() {
+		case cloudformation.ErrCodeNameAlreadyExistsException:
+			// An ErrCodeNameAlreadyExistsException occurs when a stack set already exists.
+			return true
+		}
+	}
+	return false
+}
+
+func (cf CloudFormation) waitForStackSetOperation(stackSetName, operationID string) error {
+	for {
+		response, err := cf.client.DescribeStackSetOperation(&cloudformation.DescribeStackSetOperationInput{
+			OperationId:  aws.String(operationID),
+			StackSetName: aws.String(stackSetName),
+		})
+
+		if err != nil {
+			return err
+		}
+
+		if *response.StackSetOperation.Status == "STOPPED" {
+			return fmt.Errorf("Couldn't update stack - it was stopped")
+		}
+
+		if *response.StackSetOperation.Status == "FAILED" {
+			return fmt.Errorf("Couldn't update stack - it failed")
+		}
+
+		if *response.StackSetOperation.Status == "SUCCEEDED" {
+			return nil
+		}
```

Comment:
We can use the following consts from the aws cfn package:
>  
    // StackSetOperationResultStatusPending is a StackSetOperationResultStatus enum value
    StackSetOperationResultStatusPending = "PENDING"

    // StackSetOperationResultStatusRunning is a StackSetOperationResultStatus enum value
    StackSetOperationResultStatusRunning = "RUNNING"

    // StackSetOperationResultStatusSucceeded is a StackSetOperationResultStatus enum value
    StackSetOperationResultStatusSucceeded = "SUCCEEDED"

    // StackSetOperationResultStatusFailed is a StackSetOperationResultStatus enum value
    StackSetOperationResultStatusFailed = "FAILED"

    // StackSetOperationResultStatusCancelled is a StackSetOperationResultStatus enum value
    StackSetOperationResultStatusCancelled = "CANCELLED"

Reply:
YES!